analytics_data_api/management/commands/generate_data.py

```diff
@@ -0,0 +1,339 @@
+# pylint: disable=line-too-long,invalid-name
+"""
+Abstract base management command meant to be called by other commands that generate fake data
```

Comment:
still combing through but I think this could be just free functions instead of a command object 

it's not managing state, all the generate commands already nicely take everything they need. "self" is just a decoration